pyemma/msm/ui/chapman_kolmogorov.py

```diff
@@ -127,7 +131,22 @@ def chapman_kolmogorov(dtrajs, lag, K, nsets=2, sets=None):
         C=Ccc_k.toarray()
         for l in range(len(sets)):            
             w=np.zeros(len(lcc))
-            inds = lccmap.map(sets[l])
+            """There is a problem here - the initial sets will probably be bigger than the lcc at time k*\tau"""
```

Comment:
No this is 'unfortunately' just a note that this needs a bug fix indicating where and why the problem occurs.